src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -565,12 +565,27 @@ export default class DefaultDeviceController
     return this.videoInputQualitySettings;
   }
 
-  acquireAudioInputStream(): Promise<MediaStream> {
-    return this.acquireInputStream('audio');
+  async acquireAudioInputStream(): Promise<MediaStream> {
+    if (!this.activeDevices['audio']) {
```

Comment:
Do we assign `null` to activeDevices['audio'], if yes does this code path execute say when we have selected `null` audio input device already?

Since, this is only related to Web Audio, should we call `startAudioInput(null)` explicitly under if `this.useWebAudio` is `true`? Looks like this function will execute and create empty device if current selection for audio input is `null`.

Reply:
no we only assign activeDevices['audio'].stream = <stream> from null. This is case is relevant to both webAudio is enabled/disabled (you need a dummy audio to join a call). In both case, we need empty device created first then attach it to the webAudio node if webAudio is enabled.

Reply:
Got it thanks for explanation!